05.asynchronous_sqlite3/promise.js

```diff
@@ -0,0 +1,73 @@
+import sqlite3 from "sqlite3";
+
+function run(db, sql, ...params) {
+  return new Promise((resolve) => {
+    db.run(sql, ...params, function () {
+      resolve(this);
+    });
+  });
+}
+
+function each(db, sql, callback) {
+  return new Promise((resolve) => {
+    db.each(sql, (err, row) => {
+      callback(row);
+    }),
+      () => {
+        resolve();
+      };
```

Comment:
すみません、前回見逃していたのですが、カッコの位置が変になっていないですかね？ each の引数は以下のように 3 つあるはずです。

```suggestion
    db.each(
      sql,
      (err, row) => { callback(row); },
      () => { resolve(); },
    );
```

Reply:
修正しました。
`resolve()`を括弧の内側に移動しました。